docs/server/guides/setup/minecraft-setup.md

```diff
@@ -146,6 +146,7 @@ You have your server running! To join, launch Minecraft and join the server at `
 If you want to start the server and want to be able to exit out of the terminal without the server closing, there are two ways with which you can do this
 
 - You can use a utility called [tmux](https://github.com/tmux/tmux/wiki). Use this if the server is intended to be temporarily up.
+  - You can also use `screen` which is bundled by default in debian/ubuntu distros. (see below)
```

Comment:
Indentation is a little bit off.

Screen is not bundled with Ubuntu (and also Debian afaik).

Please also add a link to screen, like the one with tmux. I.e. this one: https://www.gnu.org/software/screen/

Reply:
hm, i thought it was, i'll change

Reply:
just checked, screen **is** bundled by default on ubuntu 20.04 (at least for azure and gcp)